test/post-911-gib-status/components/EnrollmentHistory.unit.spec.jsx

```diff
@@ -14,9 +15,9 @@ const defaultProps = {
     eligibilityDate: '2016-12-01T05:00:00.000+0000',
```

Comment:
We don't seem to be using this anywhere anymore...just asked in https://github.com/department-of-veterans-affairs/vets-website/pull/6041 whether or not we should ignore it. If so, I guess we can remove it from test data as well.